packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2764,6 +2764,7 @@ export interface ImageGalleryImageProps {
 // @beta
 export interface ImageGalleryProps {
     images: Array<ImageGalleryImageProps>;
+    isOpen?: boolean;
```

Comment:
Should we make this required instead? I'm not sure what value having this as optional would be